third_party/IlmBase/eLut.h

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
please see my comments above about upstreaming OpenEXR fixes

Reply:
Agree - In the "modifications" commit, I'll remove OpenEXR and OpenVDB for the time being